bigbluebutton/greenlight
# Admin Manage Users - Active tab

## Description

- Admin Panel Manage Users tab now shows all users (needs to be changed to only show active users)
## Testing Steps

- Pull changes
- Go to Admin Panel
- Should show a list of all users for now
## Screenshots (if appropriate):

![image](https://user-images.githubusercontent.com/38328371/175584185-4e4ab93c-18ef-461b-b226-674e49a733be.png)

## TODO
Create rspec test in next PR
